tor_ocr/main.py

```diff
@@ -143,6 +143,10 @@ def decode_image_from_url(url, overlay=False, api_key=__OCR_API_KEY__):
             # crash and burn if the API is down, or similar :)
             result.raise_for_status()
 
+            if result.json()['OCRExitCode'] == 6:
+                # process timed out waiting for response
+                raise ConnectionError
```

Comment:
I definitely appreciate the comment explaining this exit code.

That said, does this mean we should re-poll the API for if it's done OCR-ing it? Or does it mean it's a 504 Gateway Timeout sort of situation and the whole OCR request needs to be run again? Does this timeout count towards our API usage or is it a freebie because the infrastructure can't handle the request?

So many questions... 😅

Reply:
I think we should just move on to the next server. The error code essentially means something went wrong on their side, and we know from experience that trying it again isn't the right option because that's what the code used to do.